ibis/bigquery/client.py

```diff
@@ -53,6 +53,16 @@ def get_table(self, table_id, dataset_id, reload=True):
     def get_schema(self, table_id, dataset_id):
         return self.get_table(table_id, dataset_id).schema
 
+    def run_statement(self, stmt):
+        query = google.cloud.bigquery.query.QueryResults(
+            stmt.replace('`', ''), self.client,
```

Comment:
I don't have context for this, but why ``stmt.replace('`', '')`` ?

[Backticks are needed around identifiers that might be reserved keywords in Standard SQL](https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#identifiers). For example, if someone has a column name called `proto`, `hash`, or `group` it would need to be enclosed in backticks.

Reply:
@tswast , something I was doing was croaking on backtick.  I forget what.  I'll try to replicate.